install.rb

```diff
@@ -1,10 +0,0 @@
-# Install hook code here
```

Comment:
I think both install and uninstall are no longer needed, what's your opinion? :)